### PR TITLE
fix: resolve init series animation on responsive

### PR DESCRIPTION
**Pull request by jungeun-cho**
### Please check if the PR fulfills these requirements
- [ ] It's submitted to right branch according to our branching model
- [ ] It's right issue type on title
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [ ] It does not introduce a breaking change or has description for the breaking change

### Description

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

File: apps/chart/src/component/heatmapSeries.ts [hunk 1]
```diff
@@ -37,6 +37,12 @@ export default class HeatmapSeries extends Component {
   render(chartState: ChartState<HeatmapChartOptions>) {
     const { layout, heatmapSeries, axes, theme, colorValueScale, options } = chartState;
 
+    this.isShow = this.visibleComponents(layout.plot);
+
+    if (!this.isShow) {
+      return;
+    }
+
```
Comment (shiren) [inline apps/chart/src/component/heatmapSeries.ts:45]: 사이즈가 없음으로 해서 안보임에 대한 처리는 개별 컴포넌트가 아니라 컴포넌트 밖에서 처리해야할 것 같아요
컴포넌트를 다루는 부분에서요.

**Review by shiren — CHANGES_REQUESTED** [review]
확인해주세요

File: apps/chart/src/component/component.ts [hunk 2]
```diff
@@ -328,4 +328,8 @@ export default abstract class Component {
       });
     }
   }
+
+  protected visibleComponents(rect) {
+    return rect.width > 0 && rect.height > 0;
+  }
```
Comment (shiren) [inline apps/chart/src/component/component.ts:334]: 이 메서드는 위치가 변경되거나 스토어의 정보든가 다른 방법을 사용하는게 좋을것 같아요..

**Comment by jungeun-cho** [comment]
@shiren @jung-han 
이 PR은 close하고 #594 로 재요청합니다.

**Pull request closed without merging**